### DISCUSSION TITLE
Shortcut for swapping output device for a program

### DISCUSSION THREAD

**Discussion by SullyPanda76cl** [File-New-Project/EarTrumpet#1233] [Ideas]
Hi, awesome program!

i've got 2 devices on my PC, enabled at the same time:
a) speaker
b) headphone

There are times where I need to swap audio output from headphone to speakers, and vice-versa, for example from web browser.
Actually I need 4 steps to do that:
a) click earTrumpet icon
b) scroll to find program
c) right click on program icon
d) click on the double arrow
e) select the other device

![ScreenShoot](https://i.imgur.com/iqJzBeb.png)

**Comment by bunch92** [comment]
There is a dev version just doing that :
https://github.com/File-New-Project/EarTrumpet/discussions/659
